src/components/Footer.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+import { Filter } from '../types/Filter';
+
+interface Props {
+  todos: Todo[];
+  filter: Filter;
+  onSelect: (filter: Filter) => void;
+  onClearCompleted: () => void;
+}
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filter,
+  onSelect,
+  onClearCompleted,
+}) => {
+  const incompleteTodoCount = todos.filter(todo => !todo.completed);
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${incompleteTodoCount.length} items left`}
```

Comment:
Add right condition for `1 item` left
```suggestion
        {`${incompleteTodoCount.length} items left`}
```